test/services/config.test.js

```diff
@@ -203,6 +203,37 @@ describe('services', () => {
       });
     });
 
+    describe('ConfigData.getCredentialsByProfileID', () => {
+      test.it('should return undefined if there are no profiles', () => {
+        const configData = new ConfigData();
+        const credentials = configData.getCredentialsByProfileID('DOES_NOT_EXIST');
+        expect(credentials).to.be.undefined;
+      });
+
+      test.it('should return error if no profiles with given profileID', () => {
+        const configData = new ConfigData();
+        configData.profiles = {};
+
+        const credentials = configData.getCredentialsByProfileID('DOES_NOT_EXIST');
+        expect(credentials.apiKey).to.equal('error');
+        expect(credentials.apiSecret).to.equal('Profile not found');
+      });
+
+      test.it('should return API keys if it profiles exists', () => {
+        const configData = new ConfigData();
+        configData.profiles = {
+          profile1: {
+            apiKey: constants.FAKE_API_KEY,
+            apiSecret: constants.FAKE_API_SECRET,
+          },
+        };
+
+        const credentials = configData.getCredentialsByProfileID('profile1');
+        expect(credentials.apiKey).to.equal(constants.FAKE_API_KEY);
+        expect(credentials.apiSecret).to.equal(constants.FAKE_API_SECRET);
+      });
+    });
+
```

Comment:
Let's add the case where credentials aren't returned from `config file`, and we'd need to fallback to `keytar`. Seems like the branch [here](https://github.com/twilio/twilio-cli-core/blob/87d963d6f4f0ece10bf2a3c47022ab8bf3f6c3bd/src/base-commands/twilio-client-command.js#L45) isn't being covered.

Reply:
Thanks for pointing this. This will need add profiles to the config file functionality. Once those changes are done for adding the profile I will use them and include the test cases accordingly.

Reply:
For the above cases we don't need addProfile functionality. But for the below cases we will need.
1. Search for the profile where profile id is stored in config file - Actual [use case](https://github.com/twilio/twilio-cli-core/pull/123/files#diff-5513dadf7e144879ca91cf0d41b25dcc6830c12c6f92bb65ebbf22cbcb63654bR224).
2. Search for the profile where profiles stored in config file but the given profile id is [not there](https://github.com/twilio/twilio-cli-core/pull/123/files#diff-5513dadf7e144879ca91cf0d41b25dcc6830c12c6f92bb65ebbf22cbcb63654bR215) in config file.